doc/api/cli.md

```diff
@@ -443,6 +443,48 @@ Affects the default output directory of:
 * [`--heap-prof-dir`][]
 * [`--redirect-warnings`][]
 
+### `--disable-warnings=code-or-type`
+
+<!-- YAML
+added: REPLACEME
+-->
+
+Disable specific process warnings by `code` or `type`.
+
+Warnings emitted from [`process.emitWarning()`][emit_warning] may contain a
+`code` and a `type`. This option will not-emit warnings that have a matching
+`code` or `type`.
+
+List of [deprecation warnings][].
+
+The Node.js core warning types are: `DeprecationWarning` and
+`ExperimentalWarning`
+
+For example, the following script will not emit
+[DEP0025 `require('node:sys')`][DEP0025 warning] when executed with
+`node --disable-warnings=DEP0025`:
+
+```js
+'use strict';
```

Comment:
Let's please always include ESM versions of code samples too, unless you can write an example that's identical for both.

We don't usually include extraneous things like `use strict` or eslint comments.

Reply:
Markdown docs failed without the eslint comment unfortunately. Not sure if there's another workaround.

Reply:
You can add `<!-- eslint-skip -->` right before the code snippet, that way it's not visible to the end users